src/contract_wrappers/registries/security_token_registry_wrapper.ts

```diff
@@ -338,6 +336,13 @@ export class SecurityTokenRegistryWrapper extends ContractWrapper {
     return tickers;
   }
 
+  /**
+   * @returns Returns the contract address
+   */
+  public async getAddress(): Promise<string> {
```

Comment:
This method is not necessary. It extends from contrac_wrapper.ts which implements 'address()' to retrieve the contract address.

Reply:
Thanks. I added it as the sandbox seemed to break without change. Will add commit to remove